# EuRoC Dataset Camera Parameters (Pinhole Model)
# Intrinsics for MH_01_easy cam0
cam_model: Pinhole
cam_width: 752
cam_height: 480
cam_fx: 458.6548807207614
cam_fy: 457.2966964634893
cam_cx: 367.2158039615726
cam_cy: 248.37534060980727
# EuRoC cam0 radial-tangential distortion (OpenCV: [k1, k2, p1, p2, k3])
cam_d0: -0.28340811217029355
cam_d1: 0.07395907389290132
cam_d2: 0.00019359502856909603
cam_d3: 1.7618711454538528e-05
cam_d4: 0.0

